foreman/data_refinery_foreman/foreman/management/commands/test_run_tximport.py

```diff
@@ -9,34 +9,34 @@
     ComputationalResult,
     ComputationalResultAnnotation,
     ComputedFile,
-    DownloaderJob,
-    DownloaderJobOriginalFileAssociation,
     Experiment,
     ExperimentOrganismAssociation,
-    ExperimentResultAssociation,
     ExperimentSampleAssociation,
     Organism,
     OrganismIndex,
     OriginalFile,
     OriginalFileSampleAssociation,
     Processor,
     ProcessorJob,
-    ProcessorJobOriginalFileAssociation,
     Sample,
     SampleResultAssociation,
 )
 from data_refinery_foreman.foreman.management.commands import run_tximport
 
 
-def run_tximport_at_progress_point(
+def prep_tximport_at_progress_point(
     complete_accessions: List[str], incomplete_accessions: List[str]
 ) -> Dict:
-    """Create an experiment and associated objects and run tximport on it.
+    """Create an experiment and associated objects that tximport needs to run on it.
 
     Creates a sample for each accession contained in either input
     list. The samples in complete_accessions will be simlulated as
     already having salmon quant run on them. The samples in
     incomplete_accessions won't.
+
+    If the accession_codes parameter is supplied the
```

Comment:
Is this comment up-to-date?

Reply:
Nope! I was going to do it like this but then I just ended up calling the different functions in the tests directly!